modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -197,7 +197,12 @@ endif::restricted[]
 ----
 <1> The base domain of the cluster. All DNS records must be sub-domains of this base and include the cluster name.
 <2> The `controlPlane` section is a single mapping, but the `compute` section is a sequence of mappings. To meet the requirements of the different data structures, the first line of the `compute` section must begin with a hyphen, `-`, and the first line of the `controlPlane` section must not. Only one control plane pool is used.
+ifndef::ibm-power[]
 <3> Specifies whether to enable or disable simultaneous multithreading (SMT), or hyperthreading. By default, SMT is enabled to increase the performance of the cores in your machines. You can disable it by setting the parameter value to `Disabled`. If you disable SMT, you must disable it in all cluster machines; this includes both control plane and compute machines.
+endif::ibm-power[]
+ifdef::ibm-power[]
+<3> Specifies whether to enable or disable simultaneous multithreading (SMT), or hyperthreading. Hyperthreading can be set by setting a value for `smtLevel`. The supported values are from 1 to 8. By default, SMT is enabled to increase the performance of the cores in your machines. You can disable it by setting the parameter value to `Disabled`. If you disable SMT, you must disable it in all cluster machines; this includes both control plane and compute machines.
```

Comment:
You cannot set it to any value between 1 and 8.  For e.g. 3, 5, 6, 7 etc.  Only Powers of 2 are allowed, 1, 2, 4, 8.